README.md

```diff
@@ -134,14 +134,14 @@ response instead.
 1. Create a `main.py` file with the following contents:
 
    ```python
-   def hello(request):
-        return "Hello world!"
+   def hello(event, context):
+        print("Received", context.event_id)
```

Comment:
I would not print `context.event_id` because it's not very useful. The message data is in `event.data`.

Reply:
The idea here is to show the messages correspond to the publisher's output, the two terminals should have the same event IDs in them.

Reply:
Ok, that sounds good. I haven't considered that.